keyboards/xelus/trinityxttkl/rules.mk

```diff
@@ -15,6 +15,5 @@ NKRO_ENABLE = yes	    # USB Nkey Rollover
 BACKLIGHT_ENABLE = no
 RGBLIGHT_ENABLE = no
 
-
```

Comment:
This seems to be a file for an unrelated board?

Reply:
Ah yes sorry. I will undo this. Was an accident